mdx2/command_line/merge.py

```
@@ -123,37 +139,36 @@ def run(args=None):
-            if args.geometry is None:
-                raise ("--geometry argument is required for symmetry-based splitting")
-            symm = loadobj(args.geometry, "symmetry")
+        elif split == "Friedel":
+            logger.info("Splitting into Friedel pairs...")
+            symm = loadobj(geometry, "symmetry")
             has_inversion = np.array([np.linalg.det(op) for op in symm.laue_group_operators]) < 0
             isminus = has_inversion[hkl.op]
             groups = [~isminus, isminus]
         else:
-            raise ("something bad happened")
+            raise Exception("something bad happened")
```

Ruff (0.14.8)

Create your own exception

(TRY002)